gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -97,6 +157,25 @@ int main(int argc, char* argv[]) {
   gematria::X86Canonicalizer canonicalizer(&llvm_support->target_machine());
   gematria::BHiveImporter bhive_importer(&canonicalizer);
 
+  llvm::exegesis::InitializeX86ExegesisTarget();
+
+  auto llvm_state_or_error = llvm::exegesis::LLVMState::Create("", "native");
+  if (!llvm_state_or_error) {
+    std::cerr << "Failed to create LLVMState\n";
```

Comment:
we don't have absl::log?

Reply:
We could, but the rest of the file uses standard library output, so I'm staying consistent with that for now. Not sure what would be preferred though. I think within Google, everything would use `absl::log`, and LLVM has its own output facilities, so this is sort of in no man's land.